content/en/_index.html

```diff
@@ -106,3 +106,16 @@ <h4>Follow us on Twitter</h4>
 			</div>
 		</div>
 </section>
+<section>
+	<div class="col">
+		<div class="row justify-content-center">
+			<div class="col-sm-12 col-md-2 col-lg-5 text-center">
+				<h4>cert-manager is a CNCF member project</h4>
+				</br>
+				<p>
+					<img src="/png/cncf.png" alt="cert-manager is a CNCF member project" width="400">
```

Comment:
Any particular reason why in the footer partial we use `{{ with resources.Get "icons/jetstack.svg" }}` and here we directly reference the png? Should they both be done in the same way?

Reply:
Looks like `{{ with resources.Get "icons/jetstack.svg" }}` embeds the svg into the HTML page, which I guess I can't do (unless I base64 encode it I guess)